### ISSUE TITLE
AlphaX: eXploring Neural Architectures with Deep Neural Networks and Monte Carlo Tree Search

### ISSUE THREAD

**Issue by Swall0w** [Swall0w/papers#444]
Wang, Linnan, Zhao, Yiyang, Jinnai, Yuu

We present AlphaX, a fully automated agent that designs complex neural
architectures from scratch. AlphaX explores the exponentially exploded search
space with a novel distributed Monte Carlo Tree Search (MCTS) and a Meta-Deep
Neural Network (DNN). MCTS intrinsically improves the search efficiency by
automatically balancing the exploration and exploitation at each state, while
Meta-DNN predicts the network accuracy to guide the search, and to provide an
estimated reward for the preemptive backpropagation in the distributed setup.
As the search progresses, AlphaX also generates the training date for Meta-DNN.
So, the learning of Meta-DNN is end-to-end. In searching for NASNet style
architectures, AlphaX found several promising architectures with up to 1%
higher accuracy than NASNet using only 17 GPUs for 5 days, demonstrating up to
23.5x speedup over the original searching for NASNet that used 500 GPUs in 4
days.

https://arxiv.org/abs/1805.07440
